WTWSMS/common/religions/00_religions.txt

```diff
@@ -1228,7 +1228,11 @@ muslim = {
 		uses_jizya_tax = yes
 	}
 	yazidi = {
-		graphical_culture = muslimgfx
+		ai_convert_same_group = 1 #Yazidis do not normally prolestyze unless asked, they tend to keep their faith to themselves, which is why they are nearing extinction in RL
```

Comment:
Good idea, but the implementation will change given the new religious system. I will put Yazidi in another category.